### ISSUE TITLE
Trigger zoom via pinch on touch display

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#578]
This should be possible on both FXL and reflow publications. In reflow mode, it would results in a font size modification. In FXL it would be a zoom on the webview, with a way to move inside the zoomed page. 

TODO: in reflow mode, study if pinch to zoom could affect images also.

**Comment by danielweck** [comment]
Related issue: https://github.com/edrlab/thorium-reader/issues/1825

**Comment by Mitia99** [comment]
@danielweck Any update on this ?

My issue is the viewer is too small when zoom is set on Fit and too big when set to 50%

![image](https://github.com/user-attachments/assets/51dc5e79-d099-418b-b826-1ea790b5d933)


![image](https://github.com/user-attachments/assets/68daf4a4-acac-488f-bdf9-aa2ccec4263b)




Thanks for adding some priority on this,

**Comment by danielweck** [comment]
related https://github.com/edrlab/thorium-reader/issues/2866
